### PR TITLE
Redirect old ssh images

### PR DESCRIPTION
**Pull request by joshmoore**
With:
- https://github.com/openmicroscopy/omero-ssh-daemon-c7-docker and
- https://github.com/openmicroscopy/omero-ssh-c7-docker

the older ome-docker images for the same purpose are no longer needed. Rather than simply delete them, this PR redirects them to the new name, effectively an alias for "latest".

### CODE REVIEW

File: omero-ssh-systemd/Dockerfile [hunk 1]
```diff
@@ -1,32 +1,3 @@
...
-RUN rm -f /lib/systemd/system/systemd*udev* ; \
-	rm -f /lib/systemd/system/getty.target;
-
-EXPOSE 22
-
-# This will automatically start systemd
-CMD ["/usr/sbin/init"]
+# Alias for omero-ssh-c7:latest
+# Likely to be renamed omero-ssh-systemd-c7
+FROM openmicroscopy/omero-ssh-c7
```
Comment (aleksandra-tarkowska) [inline omero-ssh-systemd/Dockerfile]: shouldn't that point to the tag?

Comment (joshmoore) [inline omero-ssh-systemd/Dockerfile]: Debated that, but then when the tag is bumped, we'd either have to bump this file too or let it fall behind. Seemed like having `latest` point to `latest` was safest.

**Comment by sbesson** [comment]
The aliasing strategy is nice and is probably something we could apply to other `Dockerfile` in this repo to reduce impact on whoever is consuming. Any reason not to merge?

**Comment by joshmoore** [comment]
Trying to understand where we are:

| image                                                        | dockerfile | last build  | last non-"latest" tag |
|--------------------------------------------------------------|------------|-------------|-----------------------|
| https://hub.docker.com/r/openmicroscopy/omero-ssh-c7/        | own repo   | 27 days ago | 0.1.0                 |
| https://hub.docker.com/r/openmicroscopy/omero-ssh-systemd/   | this PR    | 25 days ago | master                |
| https://hub.docker.com/r/openmicroscopy/omero-ssh-daemon-c7/ | own repo   | 25 days ago | 0.1.1                 |
| https://hub.docker.com/r/openmicroscopy/omero-ssh/           | ome-docker | 25 days ago | master                |
| https://hub.docker.com/r/openmicroscopy/omero-ssh-c6/        | ome-docker | 25 days ago | master                |
| https://hub.docker.com/r/openmicroscopy/omero-ssh-u1404/     | ome-docker | 11 days ago | -                     |

I'll try making ssh-systemd depend on `omero-ssh-c7` and then trigger builds to see if this updates correctly.

**Comment by joshmoore** [comment]
Ok. To trigger a build, I'll need to merge. The "Repository Link" has been updated. Let's see what happens.

**Pull request merged**